package.json

```diff
@@ -2,7 +2,7 @@
   "name": "t2-cli",
   "version": "0.0.9",
   "description": "The starting point of Tessel 2's command line interface",
-  "main": "./bin/tessel-2.js",
+  "main": "./index.js",
```

Comment:
The file exported to API consumers should be one that actually exports `controller` functionality.